### PR TITLE
Closes #3153 JS code for the upgrade popin toggle

### LINKED ISSUE #3153: JS for upgrade popin
Sub-task for epics #3067 #3069 

- Write JS code to manage show/hide of upgrade popin

### PR DESCRIPTION
**Pull request by Tabrisrp**
Closes #3153 

Add the JS code to toggle the upgrade popin display when clicking on the "upgrade now" button.

### CODE REVIEW

**Comment by hellofromtonya** [comment]
@Tabrisrp Can you capture a gif or vid to show this PR working? Good to see it in action without having to pull it down to validate.

**Comment by Tabrisrp** [comment]
Here you go. No styling yet for the inside of the popin, it's going to be in the next PR

JS recording

File: src/js/global/main.js [hunk 1]
```diff
@@ -159,6 +159,43 @@ $(document).ready(function(){
...
+		return false;
+	});
+
+	function wprOpenUpgradePopin(){
+		var vTL = new TimelineLite()
+			.set($wprUpgradePopin, {'display':'block'})
+			.set($wprPopinOverlay, {'display':'block'})
+			.fromTo($wprPopinOverlay, 0.6, {autoAlpha:0},{autoAlpha:1, ease:Power4.easeOut})
+			.fromTo($wprUpgradePopin, 0.6, {autoAlpha:0, marginTop: -24}, {autoAlpha:1, marginTop:0, ease:Power4.easeOut}, '=-.5')
+		;
```
Comment (hellofromtonya) [inline src/js/global/main.js:187]:
```suggestion
		var vTL = new TimelineLite();
		vTL
			.set($wprUpgradePopin, {'display':'block'})
			.set($wprPopinOverlay, {'display':'block'})
			.fromTo($wprPopinOverlay, 0.6, {autoAlpha:0},{autoAlpha:1, ease:Power4.easeOut})
			.fromTo($wprUpgradePopin, 0.6, {autoAlpha:0, marginTop: -24}, {autoAlpha:1, marginTop:0, ease:Power4.easeOut}, '=-.5')
		;
```

Comment (hellofromtonya) [reply src/js/global/main.js:187]: Here the variable is unused. A slight change to first initialize the var and then use it for the chained methods.

File: src/js/global/main.js [hunk 2]
```diff
@@ -159,6 +159,43 @@ $(document).ready(function(){
...
+	}
+
+	function wprCloseUpgradePopin(){
+		var vTL = new TimelineLite()
+			.fromTo($wprUpgradePopin, 0.6, {autoAlpha:1, marginTop: 0}, {autoAlpha:0, marginTop:-24, ease:Power4.easeOut})
+			.fromTo($wprPopinOverlay, 0.6, {autoAlpha:1},{autoAlpha:0, ease:Power4.easeOut}, '=-.5')
+			.set($wprUpgradePopin, {'display':'none'})
+			.set($wprPopinOverlay, {'display':'none'})
+		;
```
Comment (hellofromtonya) [inline src/js/global/main.js:196]: Same here.

```suggestion
		var vTL = new TimelineLite();
		
			vTL.fromTo($wprUpgradePopin, 0.6, {autoAlpha:1, marginTop: 0}, {autoAlpha:0, marginTop:-24, ease:Power4.easeOut})
			.fromTo($wprPopinOverlay, 0.6, {autoAlpha:1},{autoAlpha:0, ease:Power4.easeOut}, '=-.5')
			.set($wprUpgradePopin, {'display':'none'})
			.set($wprPopinOverlay, {'display':'none'})
		;
```

**Review by hellofromtonya — APPROVED** [review]
Approving with a minor suggestion.

**Pull request merged**